test/multipass_ex_test.exs

```diff
@@ -1,97 +1,56 @@
 defmodule MultipassExTest do
   use ExUnit.Case
   use ExUnitProperties
+
+  import MultipassEx.TestData
+
+  alias MultipassEx.{CryptoError, DecodingError, JSONDecodingError}
+
   doctest MultipassEx
 
   property "encoding and decoding" do
```

Comment:
It might be worth testing what happens when the secret/site keys are too short.

Reply:
Nothing bad (execution wise) will happen, the sha1 digest will encrypt and decrypt :ok. This whole thing isn't the best encryption/decryption setup, due to having to comply with the Ruby gem.